src/utilities/eventing/Emitter.spec.ts

```diff
@@ -0,0 +1,56 @@
+import { jest } from '@jest/globals';
+import { CloudEvent } from 'cloudevents';
+import envVar from 'env-var';
+
+import { configureMockEnvVars } from '../../testUtils/envVars.js';
+import { CE_SOURCE, K_SINK } from '../../testUtils/eventing.js';
+import { mockSpy } from '../../testUtils/jest.js';
+
+const mockEmitter = mockSpy(jest.fn());
+const mockTransport = Symbol('mockTransport');
+jest.unstable_mockModule('cloudevents', () => ({
+  emitterFor: jest.fn<any>().mockReturnValue(mockEmitter),
+  httpTransport: jest.fn<any>().mockReturnValue(mockTransport),
+}));
+// eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
Same thing as in `Kms.spec.ts`: This is a workaround until Jest fixes its EMS support and `unstable_mockModule()` is no longer needed.